insights/client/connection.py

```diff
@@ -276,7 +276,7 @@ def get_proxy(self, proxy_info, no_proxy_info, environment):
         if no_proxy_info:
             insights_service_host = urlparse(self.base_url).hostname
             logger.debug('Found NO_PROXY set. Checking NO_PROXY %s against base URL %s.', no_proxy_info, insights_service_host)
-            for no_proxy_host in no_proxy_info.split(', '):
+            for no_proxy_host in no_proxy_info.split(','):
```

Comment:
```suggestion
            for no_proxy_host in no_proxy_info.split(','):
                no_proxy_host.strip()
```
To remove any white spaces added while listing the values.

Reply:
Almost....   .strip() doesn't modify the string, it returns a new string.  Implemented as such.